assembl/static2/css/components/ideas.scss

```diff
@@ -96,3 +96,51 @@
       }
     }
 }
+
+/* iPhone Portrait */
+@media screen and (max-device-width: 480px) and (orientation:portrait) {
+  .themes-section {
```

Comment:
This change is repeating. You can make it into a mixin (not function)

http://thesassway.com/advanced/pure-sass-functions